### PR TITLE
depend-on-phpunit-cause-mock-is-deprecated

### PR DESCRIPTION
**Pull request by dominikzogg**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,10 +3,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.0",
-        "phpunit/phpunit-mock-objects": "^5.0.4|^6.0.0"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^6.5|^7.0"
+        "phpunit/phpunit": "^6.0|^7.0|^8.0"
```
Comment (TiMESPLiNTER) [inline composer.json]: @dominikzogg are you sure it still works with 6.0 like that?

Comment (dominikzogg) [reply composer.json]: i can add travis settings, but you would need to register it

Comment (TiMESPLiNTER) [reply composer.json]: I gladly will do that. 😃

Comment (dominikzogg) [reply composer.json]: done

Comment (dominikzogg) [reply composer.json]: i think 6.5 is minimal (used 6.5 in my lib for that reason)

**Pull request merged**